### ISSUE TITLE
Octoprint fails on 19.09 due to aiohttp-3.5.4 being incompatible with python2.7

### ISSUE THREAD

**Issue by otwieracz** [NixOS/nixpkgs#71940]
**Describe the bug**
Octoprint derivation does not build due to following error message:
```
error: aiohttp-3.5.4 not supported for interpreter python2.7
```

**To Reproduce**
1. Set channel to `19.09`
2. In `configuration.nix` set `services.octoprint.enable = true;`
3. Try building, will fail with `error: aiohttp-3.5.4 not supported for interpreter python2.7`

**Expected behavior**
Octoprint service starts.

**Metadata**
```
 - system: `"x86_64-linux"`
 - host os: `Linux 4.19.71, NixOS, 19.03.173619.d15a31f88a2 (Koi)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.2.2`
 - channels(root): `"nixos-19.09.907.f6dac808387"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
# a list of nixos modules affected by the problem
module:
```

**Comment by otwieracz** [comment]
From my research, it seems like Octoprint 1.3.11 began using `sentry-sdk` for version tracking, in result depending on `aiohttp` few layers deeper - and it does not support python27.

`sentry-sdk` had `calary` dependency that seemed unnecessary (at least from `pip` dependencies). Seems like removed it solved my issue.

See pull requset: https://github.com/NixOS/nixpkgs/pull/72099

**Comment by gebner** [comment]
Now it's just the werkzeug dependency, which will be fixed by #72630.

**Comment by gebner** [comment]
I've just backported the fix to the release channel.  Octoprint is now building again.
